app/src/main/java/com/celzero/bravedns/ui/SettingsFragment.kt

```diff
@@ -482,21 +507,42 @@ class SettingsFragment : Fragment(R.layout.activity_settings_screen) {
             checkForDownload(isUserInitiated = true, isRetry = true)
         }
 
-        b.settingsActivityThemeRl.setOnClickListener{
+        b.settingsActivityThemeRl.setOnClickListener {
             b.settingsActivityThemeRl.isEnabled = false
             showDialogForTheme()
+            // the count-down-timer ticks once after 500ms, at the completion of which
+            // the settingsActivityThemeRl [is re-enabled again or add whatever is appropriate.
             object : CountDownTimer(500, 500) {
-                override fun onTick(millisUntilFinished: Long) {
-                }
+                override fun onTick(millisUntilFinished: Long) {}
 
                 override fun onFinish() {
-                    if(isAdded) {
+                    if (isAdded) {
                         b.settingsActivityThemeRl.isEnabled = true
                     }
                 }
             }.start()
         }
 
+        // Ideally this property should be part of VPN.
```

Comment:
`part of VPN`...? VPN what? VpnController? VpnService? GoVpnAdapter?

Reply:
// Ideally this property should be part of VPN category / section.
// As of now the VPN section will be disabled when the
// VPN is in lockdown mode.